### ISSUE TITLE
PubSub 

### ISSUE THREAD

**Issue by HaraldEschbach** [open62541/open62541#5601]
## Background Information / Reproduction Steps
We use OPC UA server with PubSub on VxWorks and need to support security for it. 
VxWorks is using openSSL to encryption other protocols
Also open62541 has already some support for openSSL but not for PubSub.
It does not realy make sense to use two different libs on an device.

## Checklist

Please provide the following information:

 - [ ] open62541 Version 1.3:
 - [ ] Operating system: VxWorks

**Comment by jpfr** [comment]
Yes. It would be great to have PubSub security also for OpenSSL.
This hinges on these two files:

https://github.com/open62541/open62541/blob/master/plugins/crypto/mbedtls/securitypolicy_pubsub_aes128ctr.c
https://github.com/open62541/open62541/blob/master/plugins/crypto/mbedtls/securitypolicy_pubsub_aes256ctr.c

Porting them to OpenSSL should be straightforward.
Especially since most SecurityPolicy plugins are structurally the same.
Only a few key lengths and/or crypto methods from the underlying library change.

I'd estimate this requires 1-3 days of work to port.

**Comment by Muddasir-Shakil** [comment]
@jpfr 
can you add label here for help wanted and good first issues.?
